include/util.h

```diff
@@ -6,6 +6,8 @@
 #include <sys/types.h> 
 #include <xkbcommon/xkbcommon.h>
 
+#define ARRAY_SIZE(a) (sizeof(a)/sizeof(a)[0])
```

Comment:
Is that even correct?

Reply:
What would be incorrect here?

Reply:
It should be `sizeof(a[0])` instead of `sizeof(a)[0]`